app/controllers/admin/merchants_controller.rb

```diff
@@ -1,6 +1,7 @@
 class Admin::MerchantsController < ApplicationController
   def index
     @merchants = Merchant.all
+    @top_five = @merchants.top_five
```

Comment:
Just to keep with convention and could be a possible refactor later.  This method (top_five) could be called on @merchants in the view to keep the controller 'looking fit and skinny'.  
Curious what you all think about it.